src/messages.rs

```diff
@@ -249,15 +293,51 @@ impl SignedMessage {
         &self.content
     }
 
-    /// The `PublicId` associated with the signed message
-    pub fn public_id(&self) -> &PublicId {
-        &self.public_id
-    }
-
     /// The priority Crust should send this message with.
     pub fn priority(&self) -> u8 {
         self.content.priority()
     }
+
+    /// Returns whether there are enough signatures from the sender. NOTE: to ensure only validated
+    /// signatures are counted, first call `remove_invalid_signatures()`.
```

Comment:
Is there any use for counting signatures including invalid ones? Maybe it would be a good idea to just call `remove_invalid_signatures` in this function and not require callers to remember about it.